docs/content/en/docs/use-cases/hpa.md

```diff
@@ -4,18 +4,171 @@ description: Using the HorizontalPodAutoscaler
 weight: 20
 ---
 
-Use the Kubernetes Custom Metrics API
-to refer to `KeptnMetric` via the
-[Kubernetes HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
-(HPA),
-as in the following example:
+## Scaling Workloads based on Keptn metrics
+
+Kubernetes provides us with a lot of built-in capabilities to ensure
+running enough replicas in order to meet the current demand of the Workloads.
+One of these is the
+[HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+(HPA).
+
+HPA can make use of the [Keptn Metrics](../getting-started/metrics.md) custom API
+in order to scale the number of replicas of the Workloads based on the current
+load by using metrics such as CPU throttling, memory consumption or response time.
+
+### Installation of Keptn Metrics Operator
+
+To use HPA with custom metrics API, Keptn Metrics Operator needs to be installed on our cluster.
+For more information about installation please refer to the official
+[installation guide](../installation/_index.md).
+
+> **Note**
+Please be aware that Keptn Lifecycle Operator does not need to be installed for this use-case.
+
+### Installation of metrics provider (optional)
+
+If you do not have metrics provider installed on your cluster yet, please do so.
+
+For this tutorial we are going to use [Prometheus](https://prometheus.io/).
+For more information about how to install Prometheus into your cluster, please
+refer to the [official Prometheus documentation](https://prometheus.io/docs/prometheus/latest/installation/).
+
+### Deploy sample application
```

Comment:
it says so in the ticket, I think it does not hurt that we have examples how to deploy this, as there are multiple places where unexperienced users can make mistakes

Reply:
We agreed in the ticket that we will already require users to have the KeptnMetric and provider ready when starting this guide 🤷🏼

Reply:
@odubajDT and I discussed this again and we agreed that in this case it makes sense to keep the sample app and metrics + provider setup on this page, since everything needs to be referenced correctly to make sense together.